qiskit/providers/aer/backends/aer_simulator.py

```diff
@@ -323,10 +323,15 @@ class AerSimulator(AerBackend):
         shots, and a large number of qubits, with complexity around
         O(n * D^2) per shot.
 
-    * ``mps_log_data`` (str): if True, output logging data of the MPS
+    * ``"mps_log_data"`` (str): if True, output logging data of the MPS
```

Comment:
I don't see the quotes in `fusion_enable` here below

Reply:
It seems in the output that the extra quotes are not needed. I removed them here and in several other places.